File: Wizzi/ClientApp/src/dtos/CitasMedicas/registrarCitaMedicaDto.ts
export interface RegistrarCitaMedicaDto {
    codigo: string;
    codigoCliente: string;
    diagnostico: string;
    pacienteLlego: boolean;
    codigoSolicitudCitaMedica: string;
    tipoCita: string;
    activa: boolean;
    codigoSubCampaniaOrigen: string;
    codigoAgenda: string;
}

File: Wizzi/Dtos/CitasMedicas/RegistrarCitaMedicaDto.cs

namespace Wizzi.Dtos.CitasMedicas
{
    public class RegistrarCitaMedicaDto
    {
        public string codigo { get; set; }
        public string codigoCliente { get; set; }
        public string diagnostico { get; set; }
        public bool pacienteLlego { get; set; }
        public string codigoSolicitudCitaMedica { get; set; }
        public string tipoCita { get; set; }
        public bool activa { get; set; }
        public string codigoSubCampaniaOrigen { get; set; }
        public string codigoAgenda { get; set; }
    }
}
